homework-submission/react-with-json-server/src/App.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import Util from './utility.js';
+import Login from "./login-form.js";
+import AddingForm from "./add-form.js";
+import Photo from "./photo.js";
+
+
+class App extends Component {
+
+  state = {
+    userName: "no user",
+    login: false,
+    users: []
+  }
+
+  getContent = () =>{
+    Util.fetchJSON("http://localhost:4000/photos")
+      .then(users => this.setState({ users })).catch(err => console.error(err));
+  }
+
+  componentDidMount() { this.getContent() }
```

Comment:
here you are fetching the data after loading the page, means:
even if the user doesn't logged in
so why do i need to import the data from the server before log in